src/Wallabag/UserBundle/Controller/RegistrationController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Wallabag\UserBundle\Controller;
+
+use FOS\UserBundle\Controller\RegistrationController as FOSRegistrationController;
+use Symfony\Component\HttpFoundation\Request;
+
+class RegistrationController extends FOSRegistrationController
+{
+    public function registerAction(Request $request)
+    {
+        if ($this->container->getParameter('wallabag_user.registration_enabled')) {
+            parent::registerAction($request);
+        }
+        else
+        {
+            return $this->redirectToRoute('fos_user_security_login', array(), 301);
+        }
```

Comment:
Could you return the `parent::registerAction($request);` so  you can remove the else?